apps/developer-hub/src/components/EntropyTable/index.tsx

```diff
@@ -120,3 +129,80 @@ const getEntropyDeployments = async (
   const url = isMainnet ? FORTUNA_API_URLS.mainnet : FORTUNA_API_URLS.testnet;
   return await fetchEntropyDeployments(url);
 };
+
+function useEntropyFees(
+  chains: Record<string, EntropyDeployment>,
+): Record<string, bigint | undefined> {
+  const [feesByChain, setFeesByChain] = useState<
+    Record<string, bigint | undefined>
+  >({});
+  const clientsRef = useRef<Map<string, PublicClient>>(new Map());
+
+  function getClient(rpc: string): PublicClient {
+    const cached = clientsRef.current.get(rpc);
+    if (cached) return cached;
+    const client = createPublicClient({
+      transport: http(rpc, { timeout: 10_000, retryCount: 1 }),
+    });
+    clientsRef.current.set(rpc, client);
+    return client;
+  }
+
+  useEffect(() => {
+    const entries = Object.entries(chains);
+    if (entries.length === 0) return;
+
+    let isCancelled = false;
+
+    async function loadInBatches() {
+      const batchSize = 5;
+      for (let i = 0; i < entries.length; i += batchSize) {
+        if (isCancelled) return;
+        const batch = entries.slice(i, i + batchSize);
+        const results = await Promise.allSettled(
+          batch.map(async ([name, d]) => {
+            if (!d.rpc) return [name, undefined] as const;
+            try {
+              const client = getClient(d.rpc);
+              const fee = await client.readContract({
+                address: d.address as `0x${string}`,
+                abi: IEntropyV2 as unknown as Abi,
+                functionName: "getFeeV2",
+                args: [],
+              });
+              return [name, fee] as const;
+            } catch {
+              return [name, undefined] as const;
+            }
+          }),
+        );
+
+        const next: Record<string, bigint | undefined> = {};
+        for (const r of results) {
+          if (r.status === "fulfilled") {
+            const [name, fee] = r.value;
+            next[name] = fee as bigint;
+          }
+        }
+        if (Object.keys(next).length > 0) {
+          setFeesByChain((prev) => ({ ...prev, ...next }));
+        }
+      }
+    }
+
+    void loadInBatches();
```

Comment:
Personally, I really dislike using `void` for this, for a few reasons:

1. It's not clear what your intention is
2. Generally, ignoring errors is incorrect and not what you should do.

If you _really_ wanted to ignore errors, I recommend doing it like this:

```ts
loadInBatches().catch(() => { /* no-op on error */ });
```

That way, it's obvious you truly intended to disregard errors.  However, I suggest you add a proper error handler, at the very least logging the errors to the console.

Reply:
console errors are reasonable if you have nothing else in place.  Better is to use `useLogger` which is set up for you by the app shell; this does just currently do `console.error` but it centralizes the logging so we can add error reporting down the line whenever we decide to do so